0.0.1/src/client.py

```diff
@@ -150,18 +321,62 @@ def _get_next_nonce(self, proxy_account_id: str) -> int:
             logger.error(f"Failed to get next nonce: {str(e)}", exc_info=True)
             raise
 
-    async def request_signature(self, proxy_account_id: str, request: SignatureRequest) -> str:
-        """Requests signature for given parameters"""
-        logger.debug(f"Requesting signature with: {request.dict()}")
+    async def _request_multi_action_signature(
```

Comment:
This code feels pretty sensitive. Let's have a unittesting priorities list and have this at the top and we can get to it one day